packages/nl-design-system-unstable/status-badge-component/README.md

```diff
@@ -0,0 +1,81 @@
+# Status badge
+
+*Status: **In development***
+
+## Introduction
+
+A status badge draws attention and alerts a user to system states. The information in a status badge is short, concise and clearly scannable.
+
+## Variants
+
+- Success
+- Warning
+- Error
+
+## Alternatives and related components
+
+- Status light
+- Text badge
+
+## Anatomy
+
+- Container with padding
+- Standard right and bottom-margin
```

Comment:
Hmm, don't think the current implementation has standard margins? And if we need them, should they be represented in the design tokens? @Robbert

Reply:
I will remove the margin from the anatomy list!